### PR TITLE
Refactoring for translations

### PR DESCRIPTION
**Pull request by chrosey**
Refactoring HardcodedStrings into Resources and further simplified strings

Rework from #182 and #183

### CODE REVIEW

**Comment by damianstasik** [comment]
Thank you, I was fixing conflits in my pull request, now I can remove it.

**Comment by chrosey** [comment]
I had the same problem ;) needed to restart from dev..twice

File: gradle.properties [hunk 1]
```diff
@@ -1,16 +1,19 @@
...
-org.gradle.jvmargs=-Xmx1536m
+# Default value: -Xmx1024m -XX:MaxPermSize=256m
+# org.gradle.jvmargs=-Xmx2048m -XX:MaxPermSize=512m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
+#
 # When configured, Gradle will run in incubating parallel mode.
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+#Wed Jul 27 10:07:57 CEST 2016
+systemProp.http.proxyHost=10.19.241.33
```
Comment (Kolyall) [inline gradle.properties]: seems it's your system requirements

Comment (chrosey) [inline gradle.properties]: always the same... i forget to drop these while resolving conflicts....

**Comment by chrosey** [comment]
merging PRs with translations #244, #190, #228, #209

**Comment by omkarmoghe** [comment]
update to dev head pl0x

**Comment by chrosey** [comment]
no more conflicts here. or did I miss something?

**Pull request merged**